### ISSUE TITLE
no such file or directory when looking for package.json

### ISSUE THREAD

**Issue by gulfaraz** [TriPSs/conventional-changelog-action#18]
In my project, the package.json file is in a subfolder and not at the root folder.

This causes the action to fail silently with the following message,
```
(node:4861) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/runner/work/project-name/project-name/package.json'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at Object.get (/home/runner/work/_actions/TriPSs/conventional-changelog-action/v2.0.1/src/helpers/packageJson.js:13:26)
    at /home/runner/work/_actions/TriPSs/conventional-changelog-action/v2.0.1/src/index.js:27:23
```

It would be helpful if there is a possibility to either,
1. set the root folder of the project
2. set a path to the package.json file (similar to `output-file`)

**Comment by TriPSs** [comment]
Good suggestion, this is released in [v2.1.0](https://github.com/TriPSs/conventional-changelog-action/releases/tag/v2.1.0)

**Comment by gulfaraz** [comment]
Thank you for the extremely quick resolution.

Great project!
